internal/tfimport/tfimport_test.go

```diff
@@ -201,3 +202,816 @@ func TestDoesNotExist(t *testing.T) {
+				"name":      "my-helm-release",
+			},
+			"my-namespace/my-helm-release",
+		},
+		{
+			"helm_release",
+			map[string]interface{}{
+				"name": "my-helm-release",
+			},
+			"default/my-helm-release",
+		},
+		{
+			"kubernetes_config_map",
+			map[string]interface{}{
+				"metadata": map[string]interface{}{
+					"namespace": "my-namespace",
+					"name":      "my-config-map",
+				},
+			},
+			"my-namespace/my-config-map",
+		},
+		{
+			"kubernetes_config_map",
+			map[string]interface{}{
+				"metadata": map[string]interface{}{
+					"name": "my-config-map",
+				},
+			},
+			"default/my-config-map",
+		},
+		{
+			"kubernetes_manifest",
+			map[string]interface{}{
+				"manifest": map[string]interface{}{
+					"metadata": map[string]interface{}{
+						"namespace": "my-namespace",
+						"name":      "my-kubernetes-resource",
+					},
+				},
+			},
+			"my-namespace/my-kubernetes-resource",
+		},
+		{
+			"kubernetes_manifest",
+			map[string]interface{}{
+				"manifest": map[string]interface{}{
+					"metadata": map[string]interface{}{
+						"name": "my-kubernetes-resource",
+					},
+				},
+			},
+			"default/my-kubernetes-resource",
+		},
+		{
+			"kubernetes_namespace",
+			map[string]interface{}{
+				"metadata": map[string]interface{}{
+					"name": "my-namespace",
+				},
+			},
+			"my-namespace",
+		},
+		{
+			"kubernetes_pod",
+			map[string]interface{}{
+				"metadata": map[string]interface{}{
+					"namespace": "my-namespace",
+					"name":      "my-kubernetes-resource",
+				},
+			},
+			"my-namespace/my-kubernetes-resource",
+		},
+		{
+			"kubernetes_role",
+			map[string]interface{}{
+				"metadata": map[string]interface{}{
+					"namespace": "my-namespace",
+					"name":      "my-kubernetes-resource",
+				},
+			},
+			"my-namespace/my-kubernetes-resource",
+		},
+		{
+			"kubernetes_role_binding",
+			map[string]interface{}{
+				"metadata": map[string]interface{}{
+					"namespace": "my-namespace",
+					"name":      "my-kubernetes-resource",
+				},
+			},
+			"my-namespace/my-kubernetes-resource",
+		},
+		{
+			"kubernetes_service",
+			map[string]interface{}{
+				"metadata": map[string]interface{}{
+					"namespace": "my-namespace",
+					"name":      "my-kubernetes-resource",
+				},
+			},
+			"my-namespace/my-kubernetes-resource",
+		},
+		{
+			"kubernetes_service_account",
+			map[string]interface{}{
+				"metadata": map[string]interface{}{
+					"namespace": "my-namespace",
+					"name":      "my-kubernetes-resource",
+				},
+			},
+			"my-namespace/my-kubernetes-resource",
+		},
+	}
+	for _, tc := range tests {
+		importer, ok := importers[tc.resource]
+		if !ok {
+			t.Fatalf("importer does not exist for %v", tc.resource)
+		}
+
+		change := terraform.ResourceChange{Change: terraform.Change{After: tc.planFields}}
+		got, err := importer.ImportID(change, nil, false)
+		if err != nil {
+			t.Errorf("%v importer %T(%v, nil, false) returned error: %v", tc.resource, importer, change, err)
```

Comment:
You can use Fatalf here as the function just errors out

Reply:
Done.